platform/sync-installer/sync-installer-impl/src/main/java/org/codice/ddf/sync/installer/impl/SynchronizedInstallerImpl.java

```diff
@@ -541,27 +536,26 @@ private Filter getServiceIdFilter(String servicePid) throws SynchronizedInstalle
    * Covers the case where the remaining time is shorter than the poll interval. If shorter, return
    * pollInterval instead.
    */
-  private long getRemainingTime(long startTime, long maxWaitTime, long pollInterval) {
-    long remainingTime = (startTime + maxWaitTime) - System.currentTimeMillis();
-    return remainingTime < pollInterval ? pollInterval : remainingTime;
+  private long getRemainingTime(long startTime, long maxWaitTime) {
+    return (startTime + maxWaitTime) - System.currentTimeMillis();
   }
 
-  @SuppressWarnings("squid:S1149" /* Complaining about the performance const of Hashtable being synchronized. It's fine for this scenario.*/)
+  @SuppressWarnings("squid:S1149" /* required by the OSGi API. */)
   private Dictionary<String, Object> toDic(Map<String, Object> props) {
     Dictionary<String, Object> dic = new Hashtable<>();
     props.forEach(dic::put);
     return dic;
   }
 
-  private <T> Stream<T> toStream(T ele, T... eles) {
-    return Stream.concat(Stream.of(ele), Stream.of(eles));
+  private <T> Set<T> toSet(T ele, T... eles) {
+    return toStream(ele, eles).collect(Collectors.toSet());
   }
 
-  private <T> Set<T> toSet(T ele, T... eles) {
-    return Stream.concat(Stream.of(ele), Stream.of(eles)).collect(Collectors.toSet());
+  private <T> Stream<T> toStream(T ele, T... eles) {
+    return Stream.concat(Stream.of(ele), Stream.of(eles));
   }
 
-  private class SynchronizedConfigurationListener implements ConfigurationListener {
+  public static class SynchronizedConfigurationListener implements ConfigurationListener {
```

Comment:
❓ Why public? If it is just for testing, make it package private and annotate it with `@VisibleForTesting`